Gemfile

```diff
@@ -69,36 +77,35 @@ gem "axlsx", "~> 3.0.0.pre" # Write Excel files (OrganizationExports), on pre b/
 gem "wicked_pdf"
 gem "wkhtmltopdf-binary"
 
-# Rails upgrade
-gem "where-or" # backports rails 5 or query support - TODO: Rails 5 update, remove
-
 # API wrappers
 gem "simple_spark" # Sparkpost gem - we use it to send newsletters
 gem "twitter" # Twitter. For rendering tweets
 
 # OAuth provider, Grape, associated parts of API V2
 gem "api-pagination"
-gem "doorkeeper", "~> 3.1.0"
+gem "doorkeeper", "5.2.3"
 gem "grape", "~> 0.19.1"
 gem "grape-active_model_serializers", "~> 1.4.0"
 gem "grape-swagger", "0.11"
-gem "swagger-ui_rails"
+gem "swagger-ui_rails", github: "bikeindex/swagger-ui_rails", branch: "bike_index_0.1.7"
 gem "wine_bouncer"
 
 # Secure things
 gem "rack-throttle"
 gem "secure_headers", "~> 2.5.0"
 
 # Frontend
-gem "bootstrap", "~> 4.0.0.alpha3" # Bootstrap 4 - used for revised stylesheets
+gem "bootstrap", "4.0.0.alpha4" # Bootstrap 4 - used for revised stylesheets
```

Comment:
Note (so nobody needs to check what up again) - this locks to the the current version in the Gemfile.lock - it does not change the gem version.